openslides/agenda/static/js/agenda/agenda.js

```diff
@@ -401,18 +483,23 @@ angular.module('OpenSlidesApp.agenda.projector', ['OpenSlidesApp.agenda'])
     }
 ])
 
-.controller('SlideItemListCtrl', function($scope, $http, Agenda) {
-    // Attention! Each object that is used here has to be dealt on server side.
-    // Add it to the coresponding get_requirements method of the ProjectorElement
-    // class.
-    Agenda.findAll();
-    Agenda.bindAll({}, $scope, 'items');
-    $scope.ids = [];
-    var tree = $http.get('/rest/agenda/item/tree/').success(function(data) {
-        var ids = [];
-        angular.forEach(data,function(element) {
-            ids.push(element.id)
-        });
-        $scope.ids = ids;
-    })
-});
+.controller('SlideItemListCtrl', [
+    '$scope',
+    '$http',
+    'Agenda',
+    function($scope, $http, Agenda) {
+        // Attention! Each object that is used here has to be dealt on server side.
+        // Add it to the coresponding get_requirements method of the ProjectorElement
+        // class.
+        Agenda.findAll();
+        Agenda.bindAll({}, $scope, 'items');
+        $scope.ids = [];
+        var tree = $http.get('/rest/agenda/item/tree/').success(function(data) {
+            var ids = [];
+            angular.forEach(data,function(element) {
+                ids.push(element.id)
+            });
+            $scope.ids = ids;
+        })
```

Comment:
Could you use the AgendaTree function like in the ListCtrl, instead of using the tree from the server? At the moment the autoupdate wont work on this slide